zerver/models.py

```diff
@@ -460,6 +460,10 @@ class Realm(models.Model):
         default=VIDEO_CHAT_PROVIDERS["jitsi_meet"]["id"]
     )
 
+    JITSI_SERVER_SPECIAL_VALUES_MAP = {"default": ""}
+
+    jitsi_server_url: str = models.TextField(default="")
```

Comment:
What's the thinking behind having `""` be the encoding of "no URL set" rather than None/null?  Generally in the database model, we use `None/null` for this purpose.

Reply:
The rationale is somehow similar to `default_code_block_language` param. Whenever we unset the `default_code_block_language`, we pass an empty string `""` as value to `do_set_realm_property` function for updating the realm's `default_code_block_language` property value.  The reason for passing an empty string instead of `None` is that we don't update a param with a value of `None` according to the code block
`   for k, v in list(req_vars.items()):
        if v is not None and getattr(realm, k) != v:
            do_set_realm_property(realm, k, v, acting_user=user_profile)
            if isinstance(v, str):
                data[k] = "updated"
            else:
                data[k] = v` 
in `zerver.views.realm.update_realm`.

At the database level, it seems like the property value of `default_code_block_language` in the database model is always `None` if one never configure the `default_code_block_language`. On the other hand, if one sets the `default_code_block_language` to any languages (e.g. `python`) and then unset it afterwards, the property value in the database model will be an empty string `""`.

So my thought was maybe we can unify the default with an empty string `""` instead of having two different defaults, `None` and empty string `""` for `jitsi_server_url`. I am not sure about other contributors, but I used sometimes to figure out that we are having `None` and an empty string `""` as default values for `default_code_block_language`. So, I think it would be good for the development process if we can unify the default value.

Despite that, we can follow the `default_code_block_language` implementation, and we should add extra note and extra explanation (if necessary) to `default_code_block_language`'s and `jitsi_server_url`'s API definition  in `zerver/openapi/zulip.yaml` for better development experience.